tests/test_htmlchecker.py

```diff
@@ -56,6 +60,27 @@ def test_check(self):
             "0000000000000000000000000000000000000000000000000000000000000000",
         )
 
+    def test_check_with_url_template(self):
+        checker = ManifestChecker(TEST_MANIFEST_WITH_URL_TEMPLATE)
+        ext_data = checker.check()
+
+        data = self._find_by_filename(ext_data, "xdebug.tar.gz")
+        self.assertIsNotNone(data)
+        self.assertEqual(data.filename, "xdebug.tar.gz")
+        self.assertIsNotNone(data.new_version)
+        self.assertEqual(
+            data.new_version.url,
+            "https://xdebug.org/files/xdebug-2.9.0.tgz",  # noqa: E501
```

Comment:
I don't think you need to suppress E501 here

Reply:
Will this test not start failing at some point in the future when xdebug makes a new release?

Reply:
> Will this test not start failing at some point in the future when xdebug makes a new release?

No because I specifically target this release in tests/org.xdebug.Xdebug.json.

Reply:
Gotcha!